opcua/server/history.py

```diff
@@ -130,7 +310,7 @@ def event_notification(self, event):
 class HistoryManager(object):
     def __init__(self, iserver):
         self.iserver = iserver
-        self.storage = HistoryDict()
+        self.storage = HistorySQLite()  # HistoryDict() HistorySQLite()
```

Comment:
Do not overwrite the default history backend. sqlite will create a a file and we do not want to do that by default.
Create a new server_history_sql.py example